rfcs/text/0008-threat-intel.md

```diff
@@ -291,14 +310,23 @@ Stage 1: Identify potential concerns, implementation challenges, or complexity.
 -->
 
 1. How to best represent malware{name,family,type}. Current proposal is to use `threat.indicator.classification` to describe threat delivery (Hacktool etc.) and family name.
+  - This can be represented through the use of the [`threat.software.*`](https://www.elastic.co/guide/en/ecs/master/ecs-threat.html) fields.
+  - Awaiting [approval](https://github.com/elastic/ecs/pull/1480#issuecomment-889312434) of this recommendation.
```

Comment:
If we've come to an agreement, can we update or remove this line?

Reply:
Resolved.